src/components/Header.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+import DividingLine from "./DividingLine";
```

Comment:
Do you need the DividingLine just in the Header-component? Isn't it better to create the line here?

Reply:
Since I plan to use the DividingLine at some other points in my app as well, i decided to create an extra component. Otherwise that would be right and I shouldn´t need it.